### PR TITLE
Add details to aip-0001

### PR DESCRIPTION
**Pull request by stellarmagnet**

### CODE REVIEW

File: aip-0001.md [hunk 1]
```diff
@@ -7,19 +7,36 @@ Type: Process
 Created: 2022-01-23
 ```
 
-# Summary
+
+# AIP Process
+
+![AIP_process](https://user-images.githubusercontent.com/2584493/153044256-bbab0acf-05d8-42d7-9b13-29cef5dde0ce.png)
```
Comment (narodnik) [inline aip-0001.md]: This link should be https://assangedao.org/img/diagram_desktop.svg

**Review by narodnik — COMMENTED** [review]
Check out this link

File: aip-0001.md (hunk 1 above)
Comment (stellarmagnet) [reply aip-0001.md]: Doesnt render svg

Comment (narodnik) [reply aip-0001.md]: Actually don't worry since the svg won't render.

**Pull request merged**